clients/admin-ui/src/features/system/DescribeSystemStep.tsx

```diff
@@ -25,6 +26,8 @@ import {
 } from "~/features/system/system.slice";
 import { System } from "~/types/api";
 
+import { changeStep } from "../config-wizard/config-wizard.slice";
```

Comment:
I believe the import statement relative path should be "~/features/config-wizard/config-wizard.slice". Personally, I would rather use a module alias.